src/strategies/api/index.ts

```diff
@@ -13,7 +13,11 @@ export async function strategy(
   snapshot
 ) {
   let api_url = options.api + '/' + options.strategy;
-  api_url += '?network=' + network;
+  api_url += '?';
+  if (options.additionalParameters) {
+    api_url += options.additionalParameters + '&'
+  }
+  api_url += 'network=' + network;
   api_url += '&snapshot=' + snapshot;
   api_url += '&addresses=' + addresses.join(',');
```

Comment:
I would rather do that way to keep order the same everywhere (first & then param)
```suggestion
  api_url += '?network=' + network;
  api_url += '&snapshot=' + snapshot;
  api_url += '&addresses=' + addresses.join(',');
  if (options.additionalParameters) api_url += '&' + options.additionalParameters;
```

Reply:
in general it doesnt really matter.
as many APIs require stuff like ?module=example&action=whatever  i just wanted to keep the format order

i guess i also have to change the old test script to use another interface? i guess the test failed because the example used isnt online anymore